resources/prometheus/prometheus-rules.yaml

```diff
@@ -533,6 +605,11 @@ spec:
             severity: critical
             namespace: "{{ $labels.namespace }}"
             rhacs_instance_id: "{{ $labels.rhacs_instance_id }}"
+            rhacs_org_name: "{{ $labels.rhacs_org_name }}"
```

Comment:
We can't add these labels here in general, because the values originate in Central, and if Central itself is down, they don't exist.